src/test/java/bowling/FrameTest.java

```diff
@@ -0,0 +1,127 @@
+package bowling;
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package bowling.domain.frame;
```